packages/headless/src/controllers/facets/range-facet/numeric-facet/headless-numeric-facet-options.ts

```diff
@@ -26,15 +25,85 @@ import {
   numberOfValues,
 } from '../../_common/facet-option-definitions';
 
-type OptionalFacetId = Partial<
-  Pick<AutomaticRangeFacetOptions<NumericFacetRequest>, 'facetId'>
->;
+export interface NumericFacetValueOption {
+  /**
+   * The start value of the range.
+   */
+  start: number;
 
-export type NumericFacetOptions = OptionalFacetId &
-  (
-    | Omit<AutomaticRangeFacetOptions<NumericFacetRequest>, 'facetId'>
-    | Omit<ManualRangeFacetOptions<NumericFacetRequest>, 'facetId'>
-  );
+  /**
+   * The end value of the range.
+   */
+  end: number;
+
+  /**
+   * Whether to include the `end` value in the range.
+   */
+  endInclusive: boolean;
+
+  /**
+   * The current facet value state.
+   */
+  state: FacetValueState;
+}
+
+export interface NumericFacetOptions {
+  /**
+   * The field whose values you want to display in the facet.
+   */
+  field: string;
+
+  /**
+   * Whether the index should automatically create range values.
+   *
+   * Tip: If you set this parameter to true, you should ensure that the Use cache for numeric queries option is enabled for the Facet field in your index in order to speed up automatic range evaluation (see [Add or Edit Fields](https://docs.coveo.com/en/1982/index-content/add-or-edit-a-field)).
+   */
+  generateAutomaticRanges: boolean;
+
+  /**
+   * The values displayed by the facet in the search interface at the moment of the request.
+   *
+   * @default []
+   */
+  currentValues?: NumericFacetValueOption[];
```

Comment:
Since you mentioned below that this must be specified if `generateAutomaticRanges` is false, we should note that in the documentation. Also, what happens if `generateAutomaticRanges` is true? Can the user still specify `currentValues`?

Reply:
Yes, the user can still specify values.

I'll update the documentation here.